Opserver/Views/Dashboard/Dashboard.Table.cshtml

```diff
@@ -87,7 +88,7 @@ else
                                     @if (n.MemoryUsed >= 0)
                                     {
                                         <div class="progress" style="width: 215px;">
-                                            <div class="progress-bar @n.MemoryMonitorStatus().ProgressBarClass()" style="width: @(n.PercentMemoryUsed.ToString())%"></div>
+                                            <div class="progress-bar @n.MemoryMonitorStatus().ProgressBarClass()" style="width: @(n.PercentMemoryUsed.Value.ToString(CultureInfo.InvariantCulture))%"></div>
```

Comment:
I should have clarified here: `TotalMemory` can be null in some cases, that's where this would throw - fixing in a commit via `?.` approach instead (same for many others).